lwcomponent/component.go

```diff
@@ -603,3 +617,71 @@ func (c Component) EnterDevelopmentMode() error {
 
 	return nil
 }
+
+// ByVersion type to sort an array of versions by their semver comparison
+type ByVersion []semver.Version
```

Comment:
nit: This is an unusual name for a type. Is there anything more suitable than `ByVersion`

Reply:
No longer relevant since this has been removed per your next comment.